Student/StudentUITests/Tests/StudentTest.swift

```diff
@@ -49,7 +49,10 @@ class StudentTest: XCTestCase {
         // This sleep helps ensure old views got cleaned up, so EG2 doesn't find them accidentally.
         sleep(1) // FIXME: Remove this and fix flakiness better.
 
-        getApp = { return DriverFactory.getXCUITestDriver(xcuiApp!, testCase: self) }
+        getApp = {
+            // return DriverFactory.getXCUITestDriver(xcuiApp!, testCase: self)
```

Comment:
Does this need to stick around?

Reply:
We're going to be going back and forth a lot for now, so I kept it in as a comment. I tested both, and both passed.